tests/e2e/__testers__.ts

```diff
@@ -4,29 +4,68 @@ import { values } from 'lodash'
 import stripAnsi from 'strip-ansi'
 import { ClientBase } from '@prisma-spectrum/reflector/dist-cjs/Client'
 import { Reflector } from '@prisma-spectrum/reflector'
+import { getPrismaClient } from '~/tests/e2e/helpers/getPostgresAdminPrismaClient'
+import { PrismaClient } from '@prisma/client'
 
-export const testTemplate = (params: {
+export interface DBTestParams {
   templateName: PrismaTemplates.$Types.Template['_tag']
-  /** A string or regex pattern of the expected output (stdout) when the project dev mode is run.*/
   expectedDevOutput: RegExp | string
-}) => {
-  jest.setTimeout(50_000)
+  datasourceProvider: Reflector.Schema.DatasourceProviderNormalized
+  connectionStringBase: string
+  getPrismaAdmin: getPrismaClient
+  prismaConfig?: {
+    referentialIntegrity?: 'prisma' | 'foreignKeys'
+  }
+}
+
+async function dropDatabase(prismaClient: PrismaClient, databaseName: string, datasourceProvider: Reflector.Schema.DatasourceProviderNormalized) {
+  switch (datasourceProvider){
+    case 'postgres':
+      try {
+        return await prismaClient.$executeRawUnsafe(`DROP DATABASE ${databaseName} WITH (FORCE);`)
```

Comment:
Let's move the `return` to the end of the case block. Aside from there being two `return` in the case block right now it makes its also ambiguous if this is here to return a value from the function or just end the case.